koji-blame

```diff
@@ -98,20 +99,22 @@ def parse_cli(args):
     """This function parses all the arguments, options, and validates them.
     """
     parser = argparse.ArgumentParser()
-    parser.add_argument('-s', '--since', action=SinceAction, dest='since', default=EPOCH,
+    parser.add_argument('-s', '--since', action=DateAction, dest='since', default=EPOCH,
```

Comment:
Just curious, could we just put a `validate_date` function or smth (equivalent to the `__call__` function in our custom class) in for `action` instead of making a class that inherits from `argparse.Action`?  Just curious.

Reply:
I haven't tried but I think we have to inherit from `argparse.Action`